common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -306,6 +255,124 @@ def setUp(self):
         # Before every test, make sure to visit the page first
         self.advanced_settings.visit()
 
+    def test_course_author_sees_default_advanced_settings(self):
+        """
+        Test that advanced settings have the default settings
+        """
+        anonymous_discussion_setting = self.advanced_settings.get('Allow Anonymous Discussion Posts')
+        timed_exam_settings = self.advanced_settings.get('Enable Timed Exams')
+        self.assertEqual(
+            [anonymous_discussion_setting, timed_exam_settings],
+            ['true', 'false']
+        )
+
+    def test_keys_appear_alphabetically(self):
+        """
+        Test that advanced settings have all the keys in alphabetic order
+        """
+
+        key_names = self.advanced_settings.get_key_names()
+        self.assertEqual(key_names, sorted(key_names))
+
+    def test_cancel_editing_key_value(self):
+        """
+        Test that advanced settings does not save the key value, if cancel
+        is clicked from notification bar
+        """
+
+        course_display_name = self.advanced_settings.get(self.course_name_key)
+        type_in_codemirror(self.advanced_settings, 16, self.course_name_value)
+        self.advanced_settings.cancel()
+        self.advanced_settings.refresh_and_wait_for_load()
+        self.assertEqual(
+            self.advanced_settings.get(self.course_name_key),
+            course_display_name,
+            'Wrong input for Course Display Name must not change its value'
```

Comment:
Can we improve the assertion message based on what we are trying to do in the test